### ISSUE TITLE
Create scroll heatmap plugin

### ISSUE THREAD

**Issue by pa7** [pa7/heatmap.js#256]
Support for scroll heatmaps w/ a heatmap.js plugin that generates a linear gradient based on y datapoints ( ie scroll positions )
![scroll heatmap](https://cdn2.hubspot.net/hub/1951809/hubfs/Image/screen_scroll.png?width=700&height=442&t=1487966948936)

**Comment by wangdongwu** [comment]
hi,When to support?

**Comment by paulm17** [comment]
What's the ETA on this?  Thanks

**Comment by paulm17** [comment]
Thought I would bump again!  Any ETA?

**Comment by iamjoeysox** [comment]
This would be an awesome feature addition and it's the only thing holding back my company from using this in production.

**Comment by satishverma143** [comment]
yes, i also want this feature. plz let me know when this feature will come in this library.

**Comment by newyoung21** [comment]
@pa7 Hello, is this plug-in implemented?

**Comment by githubdrop** [comment]
Is this implemented

**Comment by MrMohamedAbdallah** [comment]
Any updates?

**Comment by rahulrelicx** [comment]
@pa7 any update on this?
